quizzes/migrations/0001_initial.py

```diff
@@ -1,4 +1,4 @@
-# Generated by Django 2.1.1 on 2018-09-26 23:39
+# Generated by Django 2.1.1 on 2018-10-01 23:24
```

Comment:
Won't this cause problems when this automatically deploys to Heroku? CodeShip creates a new testing database automatically each time so it won't break things there, however our deployed app will still be using the production DB, which I think would need to be removed prior to this being pushed to Heroku, though let me know if I'm wrong.

Reply:
Well it could cause a problem but nothing a good database flush won't help.

There's a reason the migration file is updated and without the updated version we won't have the appropriate fields and tables this new migration brings with it.

Reply:
Cool, I'll approve it and then we'll see what breaks 😄